extensions/objects/templates/NormalizedStringEditor.js

```diff
@@ -67,6 +69,19 @@ oppia.directive('normalizedStringEditor', [
             $scope.closeEditor();
           };
 
+          $scope.getPlaceholder = function() {
+            if (!$scope.uiConfig()) {
+              return '';
+            } else {
+              if (!$scope.uiConfig().placeholder &&
+                  DeviceInfoService.hasTouchEvents()) {
+                return $translate.instant(
+                  'I18N_PLAYER_DEFAULT_MOBILE_PLACEHOLDER');
```

Comment:
What's the rationale for using this as a default everywhere? The i18n ID seems like a player thing but this is (presumably) a component that's used in different places.

Ditto elsewhere.

Reply:
I used above function from [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/components/forms/schema_editors/SchemaBasedUnicodeEditorDirective.js#L98). It is used when user no placeholder is given. Its value is "Tap here to respond!" as can be seen [here](https://github.com/oppia/oppia/blob/develop/assets/i18n/en.json#L349).

Reply:
Hm ok. Put a note here in a comment to indicate that this reuses the same placeholder in SchemaBasedUnicodeEditor directive. Also, update the translation description in qqq.json (see assets/) to indicate that the text is a placeholder for fields to enter plain text and URLs.

Reply:
Removed this.